src/style.css

```diff
@@ -1,3 +1,103 @@
+:root {
+  --color-hover: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+}
+
+/* ul {
+  display: block;
+  list-style-type: disc;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+  padding-inline-start: 40px;
+  unicode-bidi: isolate;
+} */
+
+/* li {
```

Comment:
The code from line 27 to 31 is commented out. If this code is not needed, consider removing it to keep the code base clean.